src/assets/index.ts

```diff
@@ -151,9 +205,325 @@ const usdc_fake: AssetConfig = {
+      assetName: "WAVAX",
+      minDepositAmt: 0.001
+    },
+    "polygon": {
+      assetSymbol: "WAVAX",
+      assetName: "WAVAX (Axelar-wrapped)",
+      minDepositAmt: 0.001
+    },
+    "cosmoshub": {
+      assetSymbol: "WAVAX",
+      assetName: "WAVAX",
+      minDepositAmt: 0.0007
+    },
+    "osmosis": {
+      assetSymbol: "WAVAX",
+      assetName: "WAVAX",
+      minDepositAmt: 0.0007
+    }
+  }
+};
+
+const wmatic_polygon: AssetConfig = {
+  common_key: {
+    devnet: "wmatic-wei",
+    testnet: "wmatic-wei",
+    mainnet: "wmatic-wei",
+  },
+  native_chain: "polygon",
+  fully_supported: true,
+  decimals: 18,
+  chain_aliases: {
+    "axelar": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC",
+      minDepositAmt: 0.03
+    },
+    "moonbeam": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC (Axelar-wrapped)",
+      minDepositAmt: 0.07
+    },
+    "fantom": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC (Axelar-wrapped)",
+      minDepositAmt: 0.07
+    },
+    "ethereum": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC (Axelar-wrapped)",
+      minDepositAmt: 0.3
+    },
+    "avalanche": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC (Axelar-wrapped)",
+      minDepositAmt: 0.07
+    },
+    "polygon": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC",
+      minDepositAmt: 0.07
+    },
+    "cosmoshub": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC",
+      minDepositAmt: 0.03
+    },
+    "osmosis": {
+      assetSymbol: "WMATIC",
+      assetName: "WMATIC",
+      minDepositAmt: 0.03
+    }
+  },
+};
+
+const wftm_fantom: AssetConfig = {
+  common_key: {
+    devnet: "wftm-wei",
+    testnet: "wftm-wei",
+    mainnet: "wftm-wei",
+  },
+  native_chain: "fantom",
+  fully_supported: true,
+  decimals: 18,
+  chain_aliases: {
+    "axelar": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM",
+      minDepositAmt: 0.04
+    },
+    "moonbeam": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM (Axelar-wrapped)",
+      minDepositAmt: 0.08
+    },
+    "fantom": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM",
+      minDepositAmt: 0.08
+    },
+    "ethereum": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM (Axelar-wrapped)",
+      minDepositAmt: 0.4
+    },
+    "avalanche": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM (Axelar-wrapped)",
+      minDepositAmt: 0.08
+    },
+    "polygon": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM (Axelar-wrapped)",
+      minDepositAmt: 0.08
+    },
+    "cosmoshub": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM",
+      minDepositAmt: 0.04
+    },
+    "osmosis": {
+      assetSymbol: "WFTM",
+      assetName: "WFTM",
+      minDepositAmt: 0.04
+    }
+  },
+};
+
+const glmrName = environment === "mainnet" ? "WGLMR" : "WDEV";
+const wglmr_moonbeam: AssetConfig = {
+  common_key: {
+    devnet: "wdev-wei",
+    testnet: "wdev-wei",
+    mainnet: "wglmr-wei",
```

Comment:
Do we really need different `common_key` for each environment? I know they have different token name for `testnet` and `mainnet`, but i'm not sure that we need to have different value for the "key" as well since I think we just use `common_key` to refer to some value. 

If we don't need it to have different value, we can simply `common_key` type to be just a string.

Reply:
the common key is what we pass in for the asset identifier into the networks. so if we pass wglmr-wei for link addresses into testnet, it wouldn't work. i am told that similar situations can arise as we encounter more assets/chains, so this was a way of future proofing it. 

this isn't the case anymore, but another example was ATOM, who up until this year had been using "photon" as their lower environments assets with "atom" in mainnet